src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const peopleFilter = people.filter(person => century ? person.sex === 'm'
+  && Math.ceil(person.died / 100) === century : person.sex === 'm');
+
+  const peopleReduce = peopleFilter.reduce((sum, person) => sum
+    + (person.died - person.born), 0);
+
+  return peopleReduce / peopleFilter.length;
 }
```

Comment:
You have similar code in `calculateWomenAverageAge` function - do not repeat your self, create helper function that will calculate avarage age
```
  const peopleReduce = peopleFilter.reduce((sum, person) => sum
    + (person.died - person.born), 0);

  return peopleReduce / peopleFilter.length;
  ```